recipes/conversation_view/feed_item_row/feed_item_row.vue

```diff
@@ -157,11 +157,21 @@ export default {
       default: false,
     },
 
-    // TODO: implement as part of DT-1157
-    // state: {
-    //   type: String,
-    //   default: 'normal',
-    // },
+    /**
+     * state for the feed item row. Can be default, searched & error
+     */
+    state: {
+      type: String,
+      default: 'NORMAL',
+    },
+
+    /**
+     * Whether to fade the background color to default
+     */
+    fade: {
+      type: Boolean,
+      default: false,
+    },
```

Comment:
I am curious how this would actually be triggered on the product side, since this just triggers on mount is there a case when we might not want to fade this immediately, but rather when the specific feed item row becomes visible? After it has initially faded, there is no possible way to fade it again right? is there any case now or possibly in the future where we would need to do this?

The other possible way which might allow for more control is to just create a function that can be called via a component ref. We use this method sometimes for things like `setFocus()` and such. I think it makes a bit more sense for a "single trigger" type thing such as this.

Reply:
so i removed setTimeout since now it uses method to do the fade.